src/routes/BasketRoute/BasketRoute.test.js

```diff
@@ -0,0 +1,120 @@
+
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { MemoryRouter } from 'react-router-dom';
+import { Button } from '@folio/stripes/components';
+import { Button as ButtonInteractor } from '@folio/stripes-testing';
+import translationsProperties from '../../../test/helpers';
+import BasketRoute from './BasketRoute';
+import resources from './testResources';
+
+const AddToExistingAgreementButton = (props) => {
+  // eslint-disable-next-line react/prop-types
```

Comment:
Lets add proptypes for each of the components defined here instead of disabling lint